README.md

```diff
@@ -117,6 +117,17 @@ You can change the cache life time by passing the lifetime in minutes in the `ca
 </tabs>
 ```
 
+### Using with other libraries
+
+When using with other libraries that use the url fragment, you can disable modifying the url fragment.
+This helps using it with vue-router, or using vue-tabs-component twice in the same page.
+
+```html
+<tabs :use-url-fragment="false">
```

Comment:
I'd rather have an `options` prop dat accept an object so we can easily add more options in the future.

Reply:
I agree, changed to use an "options" object. "cache-lifetime" should be moved too.